ingests_tracker/src/main/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApi.scala

```diff
@@ -39,6 +46,7 @@ trait IngestsTrackerApi extends Runnable with Logging {
             ingestTracker.init(ingest) match {
```

Comment:
Cannot fathom why github doesn't allow comments outside of the diff, anyway - might have misunderstood the code here but does the POST endpoint not have a path while the other methods do? If not, why?

Reply:
It should have a path. It's an oversight. What is puzzling is why the tests haven't caught that.